### PR TITLE
Update windshaft configuration files for CI

### PR DESCRIPTION
**Pull request by dgaubert**
Related issue: https://github.com/CartoDB/carto-vl/issues/1411

### CODE REVIEW

**Comment by elenatorro** [comment]
I've managed to fix the docker images generation, but I'm still having problems in circle ci. There's something I'm not configuring properly when setting up the database:

Screenshot 2019-11-08 at 16 14 54

Basically, I'm getting:

```
{"mainError":{"statusCode":500,"message":"cannot connect to the database"...
```

https://circleci.com/gh/CartoDB/carto-vl/22317

It'd be great to get some help! :) @Algunenano @dgaubert

File: test/acceptance/docker/test/support/prepare_db.sh [hunk 1]
```diff
@@ -106,42 +101,9 @@ if test x"$PREPARE_PGSQL" = xyes; then
...
-      sed "s/:TESTPASS/${TESTPASS}/" |
-      sed "s/cartodb_user_de952f5f-6a86-467d-8c27-72cf6be3395e/${TESTUSER}/" |
-      sed "s/tileuser/${PUBLICUSER}/" |
-      sed "s/TO publicuser/TO ${PUBLICUSER}/" |
-      grep -v ".cdb_checkquota" |
-      PGOPTIONS='--client-min-messages=WARNING' psql -q -v ON_ERROR_STOP=1 ${TEST_DB} > /dev/null || exit 1
-  done
+  psql -c "CREATE EXTENSION IF NOT EXISTS postgis CASCADE;" ${TEST_DB}
+  psql -c "CREATE EXTENSION IF NOT EXISTS cartodb CASCADE;" ${TEST_DB}
+  psql -c "CREATE EXTENSION IF NOT EXISTS plpythonu CASCADE;" ${TEST_DB}
```
Comment (Algunenano) [inline test/acceptance/docker/test/support/prepare_db.sh]: Not related to the issue, but you don't need to install this individually. See the current Windshaft-config in:

https://github.com/CartoDB/Windshaft-cartodb/blob/master/test/support/prepare_db.sh#L73-L76

**Comment by Algunenano** [comment]
> {"mainError":{"statusCode":500,"message":"cannot connect to the database"...

This looks like the port configured in the setup (https://github.com/CartoDB/carto-vl/blob/12a4c371c491eeeffab3e74548b026125ec9f3b4/test/acceptance/docker/config/environments/test.js#L95) and the one in Docker might not match. If you have access to the docker machine, can you check what port is the database running? In debian, you can use pg_lsclusters to see all active databases and their ports.

I guess that this updates also bring a PG11 update to the image, as it's called `test-e2e-pg10` and I see:
```
 * Starting PostgreSQL 11 database server        
```
So it's likely that PG10 has the port 5432 assigned, and PG11 uses the following one (5433)

**Comment by alrocar** [comment]
I can see these errors in the postgres logs

```
2019-11-15 14:34:52.216 UTC [873] test_windshaft_cartodb_user_1@test_windshaft_cartodb_user_1_db FATAL:  role "test_windshaft_cartodb_user_1" does not exist
```

I'm going to take a look who and when creates that user in postgres and why it's not being created.

**Comment by alrocar** [comment]
I fixed some things and now I have a different error:

```
Failed to load resource: the server responded with a status of 403 (Forbidden)
[Error: CartoMapsAPIError: [Security] Unauthorized access to dataset: the provided apiKey('1234') doesn't provide access to the requested data
    at WindshaftRequestHelper._dealWithWindshaftErrors (http://localhost:5000/dist/carto-vl.js:15327:19)
    at WindshaftRequestHelper.getLayerGroup (http://localhost:5000/dist/carto-vl.js:15304:18)]
```

I've checked the requests to the Maps API are being done with the `localhost` user and api_key `1234`. The API key exists in Redis and the `database_role` has grants over the tables requested... so I don't know what's happening.

Any clue of what I could investigate?

**Comment by alrocar** [comment]
I made the Docker container to work locally, but it's still failing in CI with this error:

```
Error: TimeoutError: waiting for function failed: timeout 30000ms exceeded
```

Not sure where that timeout comes from, but could try to increase it somehow 🤷‍♂

**Comment by alrocar** [comment]
Forgot to push the changes in the Docker images to dockerhub...



but it seems we have 💚 back! \0/

**Comment by elenatorro** [comment]
I've added a section explaining how to build the docker images https://github.com/CartoDB/carto-vl/pull/1403/commits/d64cf021b895c68052f8fbd7ec71d6f4792b5aa1

Please, let me know if you think this is ok and this will be ready to merge 🚀 cc @alrocar

**Pull request merged**